extensions/amp-ad-network-fake-impl/0.1/amp-ad-network-fake-impl.js

```diff
@@ -59,15 +59,27 @@ export class AmpAdNetworkFakeImpl extends AmpA4A {
           // and the signature is "FAKESIG". This mode is only allowed in
           // `localDev` and primarily used for A4A Envelope for testing.
           // See DEVELOPING.md for more info.
-          const creative = this.transformCreativeLocalDev_(deserialized);
+          let creative = this.transformCreativeLocalDev_(deserialized);
+          try {
+            // Gracefully handle the case in which the fake ad is still
+            // formatted in JSON, which it almost certainly is, since
+            // "gulp serve" won't serve an HTML file under /extensions
+            var json = JSON.parse(deserialized);
+            if (json.creative) {
+              creative = json.creative;
+            }
+          } catch (e) {
+          }
+          // Prevent issueing a second network request to try to fetch
+          // the same JSON again.
+          this.experimentalNonAmpCreativeRenderMethod_ = XORIGIN_MODE.NAMEFRAME;
```

Comment:
@keithwrightbos This isn't allowed since this.experimentalNonAmpCreativeRenderMethod_ is private. The reason I want to do this is so that on
https://github.com/ampproject/amphtml/blob/89320dc6028af5cb0e52010df8a7df373cbf4060/extensions/amp-a4a/0.1/amp-a4a.js#L1219 it does NOT call this.renderViaCachedContentIframe_(this.adUrl_); 8 lines later.
So I would like to do an /*OK*/ to override this linter complaint. Or alternatively make a public setter for this, but that seems sketchy too. I figure you own this, per your comment on line 289 of the linked file. Thoughts?